hack/make/binary

```diff
@@ -19,6 +19,16 @@ if [ "$(go env GOOS)/$(go env GOARCH)" != "$(go env GOHOSTOS)/$(go env GOHOSTARC
 	esac
 fi
 
+if [ "$(go env GOHOSTOS)/$(go env GOHOSTARCH)" == "windows/amd64" ] && [ "$(go env GOOS)" == "windows" ]; then
+	# native compilation of Windows on Windows with golang 1.5+ needs linkmode internal
+	# https://github.com/golang/go/issues/13070
+	if [[ $LDFLAGS_STATIC_DOCKER == *"-linkmode" ]]; then
```

Comment:
Doesn't this need another `*` after the `*"-linkmode"`? (and quotes around `$LDFLAGS_STATIC_DOCKER`)